pkg/netbox/api/helper.go

```diff
@@ -24,24 +24,46 @@ import (
 	"github.com/go-openapi/strfmt"
 )
 
-type CustomFieldStringFilter struct {
-	CustomFieldName  string
-	CustomFieldValue string
+type CustomFieldEntry struct {
```

Comment:
Is `CustomField` the right wording here or should it rather be `Filter`?

Reply:
I think `CustomField` is a better name. 

This structure originated from the need to query the restoration hash stored within the custom field. 

We are leveraging this design by enabling querying multiple custom fields to support the implementation of dynamic parent prefixes.

Reply:
Makes sense.